crates/oxide/src/lib.rs

```diff
@@ -455,12 +454,10 @@ fn read_all_files(changed_content: Vec<ChangedContent>) -> Vec<Vec<u8>> {
 
 #[tracing::instrument(skip_all)]
 fn parse_all_blobs(blobs: Vec<Vec<u8>>) -> Vec<String> {
-    let input: Vec<_> = blobs.iter().map(|blob| &blob[..]).collect();
-    let input = &input[..];
-
-    let mut result: Vec<String> = input
+    let mut result: Vec<_> = blobs
         .par_iter()
-        .map(|input| Extractor::unique(input, Default::default()))
+        .flat_map(|blob| blob.par_split(|x| matches!(x, b'\n' | b'\r')))
```

Comment:
```suggestion
        .flat_map(|blob| blob.par_split(|x| matches!(x, b'\n')))
```

Might be enough to only look into `\n`, even for Windows, since otherwise every new line on Windows would emit a blank line as every line ends with both `\r` and `\n`